src/components/menu/item.tsx

```diff
@@ -34,10 +34,12 @@ const MenuItem: FC<Props> = ({
   onClick,
   url,
   active = false,
-  LinkWrapper = Fragment,
+  LinkWrapper,
 }) => {
+  const LinkWrapperComponent = LinkWrapper || Fragment
+
   return (
-    <LinkWrapper href={url}>
+    <LinkWrapperComponent {...(LinkWrapper ? { href: url } : {})}>
```

Comment:
Since URL is optional, shouldn't you check that's valid

Reply:
`href` is optional for the wrapper component too, if the `url` is not passed we will pass `{ href: undefined }` which is fine.